src/drives.js

```diff
@@ -14,6 +13,10 @@ export function getRoutesSegments(dongleId, start, end, limit) {
   return request.get(`v1/devices/${dongleId}/routes_segments`, { start, end, limit });
 }
 
+export function getRouteSegmentsById(dongleId, routeId) {
+  return StagingRequest.get(`v1/devices/${dongleId}/routes_segments?route_str=${`${dongleId}|${routeId}`.replace(/%7C/g, '|')}`);
```

Comment:
```suggestion
  return request.get(`v1/devices/${dongleId}/routes_segments?route_str=${`${dongleId}|${routeId}`.replace(/%7C/g, '|')}`);
```

can we just pass in routeName like below?